.github/workflows/build-test.yml

```diff
@@ -34,6 +34,12 @@ jobs:
       - run: yarn build
       - run: yarn lint
       - run: yarn test
+      - name: Validate RC changelog
+        if: ${{ startsWith(github.ref, 'release-v') }}
```

Comment:
Should probably change this :sweat_smile: 

Or maybe that can wait for the PR that adds the release automation

Reply:
I think the release automation actions perform this validation, so maybe this step is unnecessary?

Reply:
Huh. I thought we had actions for creating a release PR and releasing upon the merge of that PR. This step is to validate the release PR before it's merged. I didn't know that was covered in either of the release actions.

Reply:
Well, `create-release-pr` updates the changelog, and it seems to error if the changelog is invalid as well as `validateChangelog`: https://github.com/MetaMask/action-create-release-pr/blob/c220550e1faca0618d22b755dc8b1b4474473f17/src/package-operations.ts#L186

Reply:
Hm, actually maybe not. That's certainly easy to add in that Action though. Not sure if we prefer it one place over the other? Here is fine, we should just update the prefix to `release/`.

Reply:
But that doesn't get run on subsequent commits on the release PR though. This is run on all release PR commits, and the `--rc` flag additionally ensures the manual categorization is done. It doesn't sound like it does the same thing at all :thinking:

Reply:
I was referring to RC validation specifically. If we did that in `create-release-pr`, it would obviate the need for knowing the release branch prefix here. But, I'm agnostic!

Reply:
```suggestion
        if: ${{ startsWith(github.ref, 'release/') }}
```

Reply:
Right, it still doesn't sound like we're on the same page about what this actually does though. `create-release-pr` _can't_ do that, because `--rc` here also ensures that changes have been categorized. That has to be done manually _after_ the release PR has been created.

Reply:
Ah, I didn't realize it checks categorization. Thanks for clarifying!